php/class-post-type.php

```diff
@@ -1331,4 +1409,134 @@ public function handle_snapshot_fork() {
 		$forked_post['edit_link'] = get_edit_post_link( $forked_post_id, 'raw' );
 		wp_send_json_success( $forked_post );
 	}
+
+	/**
+	 * Generate resolve conflict markup.
+	 * This will add thickbox with radio button to select between conflicted setting values.
+	 *
+	 * @param string   $setting_id       setting id.
+	 * @param array    $value            setting value.
+	 * @param array    $snapshot_content snapshot post-content.
+	 * @param \WP_Post $post             Post object.
+	 */
+	public function resolve_conflict_markup( $setting_id, $value, $snapshot_content, $post ) {
+		if ( isset( $value['merge_conflict'] ) ) {
+			$setting_id_key = str_replace( ']', '\\]', str_replace( '[', '\\[', $setting_id ) );
```

Comment:
Why this? Why adding escapes?

Reply:
@westonruter 
`#TB_inline?width=600&height=550&inlineId=snapshot-post\[post\]\[1\]`
Thickbox uses jQuery, `inlineId` is selector and for `[` to work in jQuery it needs this escaping https://stackoverflow.com/a/2365005

Reply:
https://api.jquery.com/category/selectors/
>To use any of the meta-characters ( such as  !"#$%&'()*+,./:;<=>?@[\]^`{|}~ ) as a literal part of a name, it must be escaped with with two backslashes: \\.